tests/integration/test_models/test_full_battery_models/test_lead_acid/test_loqs.py

```diff
@@ -1,74 +1,68 @@
-#
-# Tests for the lead-acid LOQS model
-#
 import pybamm
 import tests
-
+import pytest
 import numpy as np
 
 
+@pytest.fixture
+def loqs_model():
+    return pybamm.lead_acid.LOQS()
```

Comment:
This looks like a redundant refactor (plus pass-by-sharing might meddle with the tests here), so I would suggest initialising the model in each test (just how it is done right now).